pkg/fees/txFees.ts

```diff
@@ -0,0 +1,17 @@
+import fetch from "node-fetch"
```

Comment:
this is in pkg/fees and the file name does not tell the user about the blockchain this is for.

Reply:
Adjusted